src/components/board/PinGalleryImg.jsx

```diff
@@ -178,9 +181,21 @@ function PinGalleryImg() {
     <StyledRoot>
       {galleryAgenda.map(({ img, title, profile, userId }, index) => {
         return (
-          <StyledGalleryImg>
+          <StyledGalleryImg key={index}>
+            <StyledDim>
+              <div>
+                <span>
+                  <h1>모든 핀</h1>
+                  <IDimMenu />
+                </span>
+                <span>저장</span>
+              </div>
+            </StyledDim>

```

Comment:
나는 비슷한 경우로 Palette 컴포넌트에서 핀을 클릭하면 회색 배경이 깔리도록 구현해야 했는데 아래 코드처럼 짰어
```suggestion
    <StyledDim>
        <StyledBackground></StyledBackground>
        <div>
            <span>
                  <h1>모든 핀</h1>
                  <IDimMenu />
                </span>
                <span>저장</span>
          </div>
                
    </StyledDim>
```
StyledDim에 position:relative 속성을 주고 Background에는 absolute를 줘서 가장 가까운 Wrapper에 달라붙도록 하는 방법이 있을듯! 참고만 해줘

Reply:
오호옹 고마워~!!